app/views/nonprofits/show.html.erb

```diff
@@ -1,2 +1,19 @@
-<h1><%= "#{@nonprofit.name}" %></h1>
+<h1><%= @nonprofit.name %>
+  <% unless @nonprofit.managers.where(user_id: current_user.id).empty? %>
```

Comment:
@ianroberts131 Line 2 works well, though its purpose is a little unclear on first reading it. 

The purpose here looks to be figuring out if the current user can edit the nonprofit.

If this is the case, consider writing a method on `User` that will reveal the purpose more clearly, say so the line could be rewritten to:

```
-  <% unless @nonprofit.managers.where(user_id: current_user.id).empty? %>
+  <% if current_user.can_edit?(@nonprofit) %>
```